CMakeLists.txt

```diff
@@ -63,7 +63,8 @@ ELSE()
     ADD_COMPILE_FLAG("-O0")
     ADD_COMPILE_FLAG("-g3")
```

Comment:
Oh, this is just throwaway code for testing on travis. Just makes the stack traces from leaks nicer to read. I'll remove when this PR is done and the leaks are gone.